service/src/main/java/de/starwit/service/jobs/AreaOccupancyRunner.java

```diff
@@ -0,0 +1,123 @@
+import jakarta.annotation.PostConstruct;
+
+@Service
+public class AreaOccupancyRunner {
+
+    private Logger log = LoggerFactory.getLogger(this.getClass());
+
+    @Value("${sae.redisStreamPrefix:output}")
+    private String REDIS_STREAM_PREFIX;
+    
+    @Value("${areaOccupancy.analyzingWindow:10s}")
+    private Duration ANALYZING_WINDOW;
+
+    @Value("${areaOccupancy.geoDistanceP95Threshold:0.001}")
+    private double GEO_DISTANCE_P95_THRESHOLD;
+    
+    @Value("${areaOccupancy.pxDistanceP95ThresholdScale:0.1}")
+    private double PX_DISTANCE_P95_THRESHOLD_SCALE;
+
+    @Autowired
+    private ObservationJobService observationJobService;
+
+    @Autowired
+    private AreaOccupancyService areaOccupancyService;
+
+    @Autowired
+    private GeoJsonService geoJsonService;
+
+    @Autowired
+    private StreamMessageListenerContainer<String, MapRecord<String, String, String>> streamListenerContainer;
+
+    private SaeMessageListener saeMessageListener;
+    
+    private List<Subscription> activeSubscriptions = new ArrayList<>();
+    private List<AreaOccupancyJob> activeJobs = new ArrayList<>();
+
+    public AreaOccupancyRunner() {
+        this.saeMessageListener = new SaeMessageListener(this::messageHandler);
+    }
+    
+    @PostConstruct
+    private void init() {
+        streamListenerContainer.start();
+        refreshJobs();
+    }
+
+    public void refreshJobs() {
+        log.debug("Refreshing jobs");
+
+        for (Subscription activeSub : activeSubscriptions) {
+            streamListenerContainer.remove(activeSub);
+        }
+        
+        this.activeSubscriptions = new ArrayList<>();
+
+        List<ObservationJobEntity> enabledJobEntites = observationJobService.findActiveAreaOccupancyJobs();
+        log.info("Enabled jobs: " + enabledJobEntites.stream().map(j -> j.getName()).collect(Collectors.joining(",")));
+
+        this.activeJobs = enabledJobEntites.stream().map(jobEntity -> new AreaOccupancyJob(jobEntity, ANALYZING_WINDOW, GEO_DISTANCE_P95_THRESHOLD, PX_DISTANCE_P95_THRESHOLD_SCALE, this::storeObservation)).toList();
+
+        for (String streamId : enabledJobEntites.stream().map(e -> e.getCameraId()).distinct().toList()) {
+            String streamKey = REDIS_STREAM_PREFIX + ":" + streamId;
+            log.info("Subscribing to " + streamKey);
+            StreamOffset<String> streamOffset = StreamOffset.create(streamKey, ReadOffset.lastConsumed());
+            Subscription redisSubscription = streamListenerContainer.receive(streamOffset, saeMessageListener);
+            activeSubscriptions.add(redisSubscription);
+        }
+    }
+
+    public void messageHandler(SaeDetectionDto dto) {
+        for (AreaOccupancyJob job : activeJobs) {
+            if (job.getConfigEntity().getCameraId().equals(dto.getCameraId())) {
+                job.addDetection(dto, Instant.now());
+            }
+        }
+    }
+
+    @Scheduled(fixedRateString = "${areaOccupancy.analyzingInterval:5s}")
```

Comment:
For some reason only PT5S is correct. Need to be also fixed in application.properties
```suggestion
    @Scheduled(fixedRateString = "${areaOccupancy.analyzingInterval:PT5S}")
```

Reply:
I have changed that to an integer and timeunit milliseconds. The friendlier parsing is a new feature in Spring Boot 3.4.0